apps/web/lib/trpc/routers/workspace.ts

```diff
@@ -52,7 +55,7 @@ export const workspaceRouter = t.router({
           actorType: "user",
           actorId: ctx.user.id,
           event: "workspace.create",
-          description: `Workspace ${input.name} created`,
+          description: `Workspace ${input.name} updated`,
```

Comment:
why?

Reply:
No idea. Do not think it was intentional.  Also noticed audit logs for name change are missing so added those.